package.json

```diff
@@ -279,6 +289,11 @@
                     "when": "view =~ /(azureResourceGroups|azureFocusView)/ && viewItem =~ /managedEnvironmentItem/i",
                     "group": "2@1"
                 },
+                {
+                    "command": "containerApps.createContainerAppFromWorkspace",
+                    "when": "view =~ /(azureResourceGroups|azureFocusView)/ && viewItem =~ /managedEnvironmentItem/i",
+                    "group": "2@2"
```

Comment:
I think create should be above delete.

Reply:
Based on talks, grouped with the existing update image command